src/dataEntryApp/views/subjectDashBoard/components/GridCommonList.js

```diff
@@ -34,7 +35,7 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const GridCommonList = ({ gridListDetails, enableReadOnly }) => {
+const GridCommonList = ({ gridListDetails, path, enableReadOnly }) => {
```

Comment:
Don't see this component using path property anywhere. So can we remove it from here and place it's being passed on from?

Reply:
Yes, will remove path from GridCommonList